pages/api/logout.ts

```diff
@@ -15,7 +15,9 @@ export async function handler(req: NextIronRequest, res: NextApiResponse<Data>)
       req.session.destroy();
       res.redirect('/');
     } catch (error) {
-      res.status(400).json({ error });
+      const errorString = JSON.stringify(error);
+      console.log(error);
```

Comment:
Adding in the `console.log` here to mirror the patterns found in other handlers, e.g. `authenticate_magic_link.ts`.